### PR TITLE
Single upcoming event to be shown

### PR DESCRIPTION
**Pull request by chotuaashiq**
#118 I changed the loadlimit size for the upcoming events to be 1.

@M-ZubairAhmed @pbteja1998 Please review.

--
### Checklist for this PR

* [ x] I have read [contributing guidelines](https://github.com/coderplex/coderplex/blob/develop/.github/CONTRIBUTING.md).
* [ x] I executed `yarn lint` locally to format my code.
* [ x] I added revelant Tests to verify my code.
* [ x] All tests `yarn test` ran successfully.
* [ x] Pull request is towards `develop` branch.
* [ x] I have tested responsiveness of the UI by checking on devices using [ScreenFly](http://quirktools.com/screenfly/) 
* [ x] Added myself to contributors table 
* [ x] Documentation
* [ x] Ready to be merged

### CODE REVIEW

File: pages/events.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default class Events extends React.Component {
     pastEvents: [],
     pastEventsLoadLimit: 2,
     futureEvents: [],
-    futureEventsLoadLimit: 2,
+    futureEventsLoadLimit: 1,
```
Comment (M-ZubairAhmed) [inline pages/events.js]: We arent looking at this to change.

**Comment by chotuaashiq** [comment]
@M-ZubairAhmed I do not ReactJS so I am not able to get how to use that in the index.js file. Can you tell me what I should do.

**Comment by M-ZubairAhmed** [comment]
@chotuaashiq its inspiring that you would want to contribute, i would suggest you go through this [ReactJS](https://github.com/coderplex/learn/blob/master/web-dev/Frontend/Libraries%20%26%20Frameworks/Learn-React.md#components) and look for components section. We have prepared a curated list of all react best learning resource. Also there is a [webcast](https://www.meetup.com/coderplex/events/245827263/) today where we are going to explain our code base. I would like to invite you to that and you can ask lots of questions there.

**Comment by M-ZubairAhmed** [comment]
ping!

**Comment by vinaypuppal** [comment]
Closing this for #133

**Pull request closed without merging**